INSTALLATION.md

```diff
@@ -102,12 +102,36 @@ If all goes well, a new ROM should be built at `build/hackeroot-mq/hackeroot-mq.
 
 Both of these have the disadvantage that the ordering of the terminal output is scrambled, so for debugging it is best to stick to one thread (i.e. not pass `-j` or `-jN`).
 
-#### 6. Creating a BPS patch
+#### 6. Other targets
 
+##### Compressing the rom
+To compress the rom, run the following command:
+
+```bash
+make compress
+```
+
+##### Building for Wii VC
+To build a .wad for Wii VC, run the following command:
+
+```bash
+make wad
+```
+
+**NOTE:** this is assuming you followed the instructions for Wii VC in [step 3](#3-prepare-a-base-rom).
+
+##### Creating a BPS patch
 To generate a .bps patch you can run the following command:
 
 ```bash
 make patch
 ```
 
 The patch will be in the ``build/hackeroot-mq/`` folder.
+
+#### 7. Toggle HackerOoT Features
+
+You can customize which HackerOoT features you want in your project and which not in the `include/config/` folder.
+Just define the features as `true` or `false` or, if necessary, put in the value you want (e.g. Text speed).
+
+**TIP:** if you are using VS Code, look at [vscode.md](docs/vscode.md) for useful costumization that make working with this project easier.
```

Comment:
"costumization"

Reply:
i meant like costumizing vscode for the project, couldnt think of a better word

Reply:
isn't it "customization"? I thought you did a small typo here

Reply:
ah, didn't even notice that
fixed now